src/astro/files/type/ndjson.py

```diff
@@ -0,0 +1,59 @@
+import io
+import json
+from typing import Union
+
+import pandas as pd
+
+from astro.constants import DEFAULT_CHUNK_SIZE
+from astro.constants import FileType as FileTypeConstants
+from astro.files.type.base import FileType
+
+
+class NdJson(FileType):
```

Comment:
Either we name it `Ndjson` or `NDJSON` (New Delimited JavaScipt Object Notation),

Reply:
Went with - NDJSON